testing/upload_validation.ts

```diff
@@ -274,6 +274,15 @@ export function validateCrawlHierarchy(
           return undefined;
         }
 
+        if (inheritPermissions && tableSchema.idProperty !== propertyKey) {
```

Comment:
If we want multi level id inheritance we would need to remove this

Reply:
Validation would be harder in that case as we would need to understand what it comes from